### PR TITLE
console=>log

### PR DESCRIPTION
**Pull request by larshp**

### CODE REVIEW

**Comment by larshp** [comment]
first part = https://github.com/open-abap/open-abap-core/pull/518

**Comment by larshp** [comment]
awaiting approval by @exercism/maintainers-admin

**Review by ErikSchierboom — APPROVED** [review]

**Review by iHiD — APPROVED** [review]
Looks good to my eyes.

**Pull request merged**
